app/src/main/java/com/github/se/orator/ui/offline/OfflineInterviewModule.kt

```diff
@@ -0,0 +1,128 @@
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import com.github.se.orator.model.symblAi.SpeakingViewModel
+import com.github.se.orator.ui.navigation.NavigationActions
+import com.github.se.orator.ui.navigation.Screen
+import com.github.se.orator.ui.theme.AppColors
+import com.github.se.orator.ui.theme.AppDimensions
+import com.github.se.orator.ui.theme.AppFontSizes
+import kotlin.random.Random
+import savePromptsToFile
+
+fun generateRandomString(length: Int = 8): String {
+  val charset = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789"
+  return (1..length).map { charset[Random.nextInt(charset.length)] }.joinToString("")
+}
+
+/**
+ * The SpeakingJobInterviewModule composable is a composable screen that displays the job interview
+ * module.
+ *
+ * @param navigationActions The navigation actions that can be performed.
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@SuppressLint("SuspiciousIndentation")
+@Composable
+fun OfflineInterviewModule(
+    navigationActions: NavigationActions,
+    speakingViewModel: SpeakingViewModel
+) {
+  var targetCompany by remember { mutableStateOf("") }
+  var jobPosition by remember { mutableStateOf("") }
+  val ID = generateRandomString()
+
+  val context = LocalContext.current
+
+  val inputFields =
+      listOf(
+          InputFieldData(
+              value = targetCompany,
+              onValueChange = { newValue: String -> targetCompany = newValue },
+              question = "What company are you applying to?",
+              placeholder = "e.g Apple, Google",
+              testTag = "company"),
+          InputFieldData(
+              value = jobPosition,
+              onValueChange = { newValue: String -> jobPosition = newValue },
+              question = "What job position are you applying for?",
+              placeholder = "e.g Hardware engineer",
+              testTag = "jobInput"))
+
+  Column(
+      modifier =
+          Modifier.fillMaxSize()
+              .padding(horizontal = AppDimensions.paddingMedium)
+              .padding(top = AppDimensions.paddingSmall)
+              .verticalScroll(rememberScrollState())
+              .testTag("content"),
+      verticalArrangement = Arrangement.Center) {
+        inputFields.forEach { input ->
+          OutlinedTextField(
+              value = input.value,
+              onValueChange = input.onValueChange,
+              label = { Text(input.question, color = AppColors.textColor) },
+              placeholder = { Text(input.placeholder, color = AppColors.textColor) },
+              modifier = Modifier.fillMaxWidth().testTag(input.testTag),
+              colors =
+                  TextFieldDefaults.outlinedTextFieldColors(
```

Comment:
I think you should consider using Material colors here to maintain consistency with the app's theme and ensure a matching look across all UI components!